kube/service.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: little-help-book
+
+  namespace: little-help-book
+  annotations:
+    getambassador.io/config: |
+      ---
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: stage-little-help-book-mapping
+      host: mvpstudio.stage.apps.mvpstudio.org
```

Comment:
This would have `http://mvpstudio.stage.apps.mvpstuido.org/api` serve the little help book. Needs to be the URL you actually want.

Reply:
PR#3